spec.html

```diff
@@ -39692,13 +39701,32 @@ <h1>Memory Model Fundamentals</h1>
       </table>
     </emu-table>
 
+    <p>A <dfn>Synchronize event</dfn> is represented by a <dfn>Synchronize</dfn> Record.</p>
+
+    <emu-table id="table-synchronize-fields" caption="Synchronize Event Fields">
+      <table>
+        <tbody>
+          <tr>
+            <th>Field Name</th>
+            <th>Value</th>
+            <th>Meaning</th>
+          </tr>
+          <tr>
+            <td>[[Order]]</td>
+            <td>`"SeqCst"`</td>
```

Comment:
Is this intended future-proofing for release-acquire?

The reason RMW events have a fixed [[Order]] field is because they're quantified over along with other Shared Data Blocks that refer to [[Order]] generically. Synchronize events don't participate in that stuff, so we could leave it out for simplicity.

Reply:
Partially this, and also to typographically give Synchronize events their own table structure on par with SDB events. I've changed to a textual description, which doubles as an explanatory note.